ramalama/common.py

```diff
@@ -26,19 +26,30 @@
 DEFAULT_IMAGE = "quay.io/ramalama/ramalama"
 
 
+_engine = ""
+
+
 def container_manager():
+    global _engine
+    if _engine != "":
+        if _engine == "None":
+            return None
```

Comment:
Empty string, "None" and None... Python is frustrating with all the various kinds of null :'(